kahuna/public/js/image/crop.html

```diff
@@ -15,4 +15,11 @@
     <span class="flex-spacer"></span>
     <span class="image-crop__creator" title="Cropped by {{:: crop.author}} at {{:: crop.date | date:'medium'}}">{{:: crop.author | getInitials}}</span>
   </div>
+  <asset-handle
+    data-source="grid"
+    data-source-type="crop"
+    data-thumbnail="{{:: extremeAssets.smallest | assetFile}}"
+    data-primary-id="{{ctrl.image.data.id}}"
+    data-secondary-id="{{crop.id}}"
```

Comment:
What's the reason for the split ID here? What do we use this for? In my head we need these to drop the right thing into the Composer editor. The editor understands URIs, so can we use crop URIs here? Otherwise something presumably needs to know how to make the URI anyway?

Reply:
correct, we build the embed URL in pinboard, however in the sprit of your original comment/suggestion we should probably have an `data-embed-url` rather than the primary & secondary data attributes... problem is, I'm struggling to find the code that sets the browser URL when a crop is selected (as its that URL which we're building on the pinboard side, given thats what composer supports)

Reply:
ok I managed to work it out, what do you think of... https://github.com/guardian/grid/pull/3190/commits/2f5c4a8dbc58422be6b696c12409e154072d85a0